specification/botservice/resource-manager/Microsoft.BotService/stable/2020-06-02/examples/CreateBot.json

```diff
@@ -0,0 +1,105 @@
+{
+  "parameters": {
+    "subscriptionId": "subscription-id",
+    "resourceGroupName": "OneResourceGroupName",
+    "api-version": "2017-01-01",
+    "resourceName": "samplebotname",
+    "parameters": {
+      "location": "West US",
+      "sku": {
+        "name": "S1"
+      },
+      "etag": "etag1",
+      "tags": {
+        "tag1": "value1",
+        "tag2": "value2"
+      },
+      "kind": "sdk",
+      "properties": {
+        "description": "The description of the bot",
+        "developerAppInsightKey": "appinsightskey",
+        "developerAppInsightsApiKey": "appinsightsapikey",
+        "developerAppInsightsApplicationId": "appinsightsappid",
+        "displayName": "The Name of the bot",
+        "endpoint": "http://mybot.coffee",
+        "iconUrl": "http://myicon",
+        "luisAppIds": [
+          "luisappid1",
+          "luisappid2"
+        ],
+        "luisKey": "luiskey",
+        "msaAppId": "exampleappid"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "location": "West US",
+        "tags": {
+          "tag1": "value1",
+          "tag2": "value2"
+        },
+        "name": "samplename",
+        "type": "sampletype",
+        "id": "someid",
+        "kind": "sdk",
+        "etag": "etag1",
+        "properties": {
+          "description": "The description of the bot",
+          "developerAppInsightKey": "appinsightskey",
+          "developerAppInsightsApplicationId": "appinsightsappid",
+          "displayName": "The Name of the bot",
+          "endpoint": "http://mybot.coffee",
+          "endpointVersion": "version",
+          "iconUrl": "http://myicon",
+          "luisAppIds": [
+            "luisappid1",
+            "luisappid2"
+          ],
+          "msaAppId": "msaappid",
+          "configuredChannels": [
+            "facebook",
+            "groupme"
+          ],
+          "enabledChannels": [
+            "facebook"
+          ]
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "location": "West US",
+        "tags": {
+          "tag1": "value1",
+          "tag2": "value2"
+        },
+        "name": "samplename",
+        "type": "sampletype",
+        "id": "someid",
```

Comment:
Id should be something of the form `/subscriptions/{id}/resourceGroups/{rg}/providers/Microsoft.BotService/botServices/samplename`

I would also recommend updating all the examples to remove this issue (type and id will vary by resource type, of course).

Reply:
Did this. Updated api-version as well.